packages/lesswrong/components/ea-forum/giving-portal/ElectionFundCTA.tsx

```diff
@@ -62,6 +63,7 @@ const styles = (theme: ThemeType) => ({
     borderRadius: theme.borderRadius.small,
     padding: 16,
     outline: "none",
+    whiteSpace: "nowrap",
```

Comment:
Separate fix for the text wrapping. This does make the rightmost one a bit wider than the others, but only on a very narrow range of screen sizes so I think this is better